pynipap/pynipap.py

```diff
@@ -147,7 +147,12 @@
     Classes
     -------
 """
-import xmlrpclib
+try:
+    import xmlrpc.client as xmlrpclib
+except ImportError:
+    import xmlrpclib
+    int = long
```

Comment:
Somewhat weird not being able to actually use int after this.. but I guess it's ok. I prefer py3 "correctness" in code (ie using int) than being closer to py2.